packages/graphql/lib/schema.tsp

```diff
@@ -0,0 +1,23 @@
+import "../dist/src/lib/schema.js";
+
+using TypeSpec.Reflection;
+
+namespace TypeSpec.GraphQL;
+
+namespace Schema {
```

Comment:
Would this namespace result in any conflicts?

Reply:
No, because it is itself namespaced. The full namespace is `TypeSpec.GraphQL.Schema`.